patches/tModLoader/Terraria/Localization/GameCulture.TML.cs

```diff
@@ -0,0 +1,21 @@
+using System.IO;
+using System.Linq;
+
+namespace Terraria.Localization;
+
+public partial class GameCulture
+{
+	public static GameCulture FromPath(string path) {
+		path = Path.ChangeExtension(path, null);
+
+		string[] split = path.Split("/");
```

Comment:
Probably should be using `Path.DirectorySeparatorChar`

Reply:
(It doesn't work here, path contains "/" while Path.DirectorySeparatorChar is "\\" (at least on Windows)